src/exercise/02.html

```diff
@@ -4,25 +4,18 @@
 <body>
   <div id="root"></div>
 
-  <!-- 💣 remove this line
   <script src="https://unpkg.com/react@17.0.0/umd/react.development.js"></script>
   <script src="https://unpkg.com/react-dom@17.0.0/umd/react-dom.development.js"></script>
-  💣 remove this line -->
 
   <script type="module">
-    const rootElement = document.getElementById('root')
+    const rootElement = document.getElementById('root');
+    const hello = React.createElement('span', null, 'Hello');
+    const world = React.createElement('span', null, 'World');
 
-    // You're going to re-implement this code using React!
-    // 💣 So go ahead and delete this implementation (or comment it out for now)
-    // These three lines are similar to React.createElement
-    const element = document.createElement('div')
-    element.textContent = 'Hello World' // 💰 in React, you set this with the "children" prop
-    element.className = 'container' // 💰 in React, this is also called the "className" prop
-    // This is similar to ReactDOM.render
-    rootElement.append(element)
-
-    // 🐨 Please re-implement the regular document.createElement code above
-    // with these React API calls
-    // 💰 The example in the markdown file should be a good hint for you.
+    const helloWorld = React.createElement('div', {
+      className: 'container',
+      children: [ hello, ' ', world ],
```

Comment:
I skipped the space in mine. The markup in the exercise just had the two spans so 🤷  😆